docker/run.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -e
+
+BUILDER_VOLUME="$(dirname $PWD)"
+CACHER_VOLUME="$HOME/apt_cacher_mnt"
+IMG="derivative-maker/derivative-docker"
+USER="user"
+LOG_DIR="$PWD/logs"
+[ -d ${LOG_DIR} ] || mkdir -p ${LOG_DIR}
+
+sudo modprobe -a loop dm_mod
+
+sudo docker run --name derivative-docker -it --rm --privileged \
+	--env "TAG=17.4.0.3-developers-only" \
+ 	--env 'flavor_meta_packages_to_install=' \
+	--env 'install_package_list=' \
+	--env 'DERIVATIVE_APT_REPOSITORY_OPTS=' \
+	--volume ${BUILDER_VOLUME}:/home/user \
+	--volume ${CACHER_VOLUME}:/var/cache/apt-cacher-ng ${IMG} \
+	/bin/bash -c  "/usr/bin/su ${USER} --preserve-environment --session-command '/usr/bin/start_build.sh \
+	--flavor whonix-gateway-cli \
+	--target qcow2 \
+	--type vm \
+	--arch amd64 \
```

Comment:
All of these would be good to avoid hardcoding if possible. Perhaps this script could take `--flavor`, `--target`, `--type`, and `--arch` arguments which would then be passed through? `--repo` might also be good to avoid hardcoding. (Not a blocker for merging this, IMO.)